[Docs] add set command examplee to sql api (#18027)
apache/druid: docs/api-reference/sql-ingestion-api.md

### Sample request

The following example shows a query that fetches data from an external JSON source and inserts it into a table named `wikipedia`. It specifies two query context parameters:

> - `maxNumTasks=3`: This limits the max number of parallel tasks for this data loading job to 3.

Reviewer: ```suggestion
- `maxNumTasks=3`: This limits the maximum number of parallel tasks to 3.
```

> - `maxNumTasks=3`: This limits the maximum number of parallel tasks to 3.
> - `finalizeAggregations=false`: This prevents Druid from performing final data aggregation during loading. It's useful if you want to work with intermediate data states or control aggregation later. For more information, see [Rollup](../multi-stage-query/concepts/#rollup).

Reviewer: ```suggestion

- `maxNumTasks=3`: Limits the maximum number of parallel tasks to 3.
- `finalizeAggregations=false`: Prevents Druid from performing final data aggregation during loading. It's useful if you want to work with intermediate data states or control aggregation later. For more information, see [Rollup](../multi-stage-query/concepts.md#rollup).
```

> - `finalizeAggregations=false`: This prevents Druid from performing final data aggregation during loading. It's useful if you want to work with intermediate data states or control aggregation later. For more information about Rollup, see [Rollup](../multi-stage-query/concepts/#rollup).

Reviewer: ```suggestion
- `finalizeAggregations=false`: This prevents Druid from performing final data aggregation during loading. It's useful if you want to work with intermediate data states or control aggregation later. For more information, see [Rollup](../multi-stage-query/concepts/#rollup).
```

### Sample request

> The following example shows a query that fetches data from an external JSON source and inserts it into a table named `wikipedia`.It specifies two special settings directly in the query before inserting the data:

Reviewer: ```suggestion
The following example shows a query that fetches data from an external JSON source and inserts it into a table named `wikipedia`. It specifies two query context parameters:
```

### Sample request

The following example shows a query that fetches data from an external JSON source and inserts it into a table named `wikipedia`.
The example specifies two query context parameters:

- `maxNumTasks=3`: Limits the maximum number of parallel tasks to 3.

> - `finalizeAggregations=false`: Prevents Druid from performing final data aggregation during loading. It's useful if you want to work with intermediate data states or control aggregation later. For more information, see [Rollup](../multi-stage-query/concepts.md#rollup).

Author: ```suggestion
- `finalizeAggregations=false`: Ensures Druid saving intermediate data aggregation during loading. It's useful if you want to work with intermediate data states or control aggregation later. For more information, see [Rollup](../multi-stage-query/concepts.md#rollup).
```
